platform/installation.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Platform Installation
+summary: Guidance on the different ways to install and use the platform
+reviewed: 2019-06-15
+---
+
+The Particular Service Platform can be used in different ways.
+
+- Monitoring production systems
+- Debugging during development
+- Presenting from within Visual Studio
+
+## Monitoring production systems
+
+The Platform Installers requires internet connectivity and performs several actions like verifying MSMQ and MSDTC installation. It is possible to [achieve the same result](/platform/installer/offline.md) without requiring internet connectivity.
```

Comment:
```suggestion
The Platform Installer requires Internet connectivity. It is possible to [achieve the same result](/platform/installer/offline.md) without requiring internet connectivity.
```

Reply:
I don't believe it's necessary to muddy the waters about MSMQ stuff since it's totally optional.